web/packages/core/texts/ja-JP/messages.ftl

```diff
@@ -16,6 +16,14 @@ view-error-details = エラーの詳細を表示
 open-in-new-tab = 新しいタブで開く
 click-to-unmute = クリックでミュートを解除
 clipboard-message-title = コピーしてRuffleに貼り付け
+clipboard-message-description =
+    { $variant ->
+       *[unsupported] お使いのブラウザはクリップボードへのフルアクセスをサポートしていません。
+        [access-denied] クリップボードへのアクセスが拒否されました。
+    } ですが、以下のショートカットを利用できます:
+clipboard-message-copy = { " " } からコピー
+clipboard-message-cut = { " " } : 切り取り
+clipboard-message-paste = { " " } : 貼り付け
```

Comment:
It's strange that these are not consistent...

Reply:
This is my mistake, sorry.
I will correct it later.